src/responses.rs

```diff
@@ -46,3 +46,49 @@ impl Default for VerifyPoStResponse {
         }
     }
 }
+
+///////////////////////////////////////////////////////////////////////////////
+/// VerifyPieceInclusionProofRespone
```

Comment:
TODO: Typo. `VerifyPieceInclusionProofRespone` should be `VerifyPieceInclusionProofResponse`.